sympy/plotting/plot.py

```diff
@@ -108,6 +104,68 @@ def new_bound(t, bound):
     return args
 
 
+def _build_line_series(*args, **kwargs):
+    """Loop over the provided arguments and create the necessary line series.
+    """
+    series = []
+    sum_bound = int(kwargs.get("sum_bound", 1000))
+    for arg in args:
+        expr, r, label, rendering_kw = arg
+        kw = kwargs.copy()
+        if rendering_kw is not None:
+            kw["rendering_kw"] = rendering_kw
+        # TODO: _process_piecewise check goes here
+        if not callable(expr):
+            arg = _process_summations(sum_bound, *arg)
+        series.append(LineOver1DRangeSeries(*arg[:-1], **kw))
+    return series
+
+
+def _create_series(series_type, plot_expr, **kwargs):
+    """Extract the rendering_kw dictionary from the provided arguments and
+    create an appropriate data series.
+    """
+    series = []
+    for args in plot_expr:
+        kw = kwargs.copy()
+        if args[-1] is not None:
+            kw["rendering_kw"] = args[-1]
+        series.append(series_type(*args[:-1], **kw))
+    return series
+
+
+def _set_labels(series, labels, rendering_kw):
```

Comment:
Back-compatibility.

A series is used to generate numerical data and store a few style-related attributes, like `label`, or `line_color`, or `surface_color` inherited from the old implementation, and now the more general `rendering_kw`.

 After a series has been created, sooner or later it will be processed by a renderer: it is just a function which extract the numerical data and styling options from the series and adds them to the chart.

Reply:
No. `Series` was more simple before https://github.com/sympy/sympy/pull/25469, and you started to add a bunch of things there.
Well, I start to believe that `label` or `line_color` makes sense. 
Because `line_color` can be similar to 'vertex colors' which is often nontrivial with geometry
And label makes sense in some regards because there is no big deal to consider that as 'name' of a geometry.
But I didn't have a roadmap for that to extend to have much more attributes that is not always necessary for them.

Reply:
I read your comment 3 times, and I still don't understand what you are trying to say?!?!?!?!?!

Reply:
Don’t be personal
It does not intimidate me.

Reply:
@sylee957 Please, consider writing in your native language. Then, I'll use Google Translate or something similar. Because right now, I don't understand what kind of grammar you are using. At least 3 out of 4 of your sentences doesn't make any sense to me... It's like reading a text generated from a really old chat-bot: I don't understand you! So, don't take it personal. A communication is successful if two or more people understands each other. I'm unable to understand what you are writing.

Reply:
> Please, consider writing in your native language. Then, I'll use Google Translate or something similar. Because right now, I don't understand what kind of grammar you are using. At least 3 out of 4 of your sentences doesn't make any sense to me... It's like reading a text generated from a really old chat-bot:

I have web archives of your comments